tests/queries/0_stateless/00398_url_functions.sql.j2

```diff
@@ -177,3 +177,12 @@ SELECT cutQueryStringAndFragment('http://paul@www.example.com/a/b/c?a=b#d=f');
 SELECT cutQueryStringAndFragment('//paul@www.example.com/a/b/c?a=b#d=f');
 SELECT cutQueryStringAndFragment('//paul@www.example.com/a/b/c#d=f');
 
+SELECT '====RESOLVE RELATIVE URL====';
+SELECT resolveRelativeURL('image.gif', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('image.gif', 'https://clickhouse.com/blog');
```

Comment:
```sql
SELECT resolveRelativeURL('image.gif', 'https://clickhouse.com/blog');
```
has result `https://clickhouse.com/image.gif`. Why not `https://clickhouse.com/blog/image.gif`? Perhaps that is intended but then the documentation does not really explain how the resolving algorithm works.

Reply:
it is my mistake, i will fix it

Reply:
this behavior is the same as urllib.parse.join in python3, i think this package already implemented with the RFC, so i chose to do how python3 reacts

Reply:
@rschu1ze, it should behave exactly as the browser does, when parsing HTML pages.
This behavior is correct.